### PR TITLE
Dont copy SQL DB creation scripts into executable

### PR DESCRIPTION
**Pull request by whyman**
This change stops generation of zlib compressed blobs in headers
containing the initialisation SQL for the 2 DB types.

Instead we simply load it off of the filesystem.

Introduces two new config options:
/storage/sqlite|mysql/init-sql-path which allows overriding the default path.

Default path is `datadir/mysql.sql` and `datadir/sqlite3.sql` respectively.

Fixes cross compilation issues caused by trying to run a executable we had
just built with a non-native compiler at build time.

Fixes: #1156

### CODE REVIEW

File: src/database/mysql/mysql_database.cc [hunk 1]
```diff
@@ -203,30 +194,18 @@ void MySQLDatabase::init()
...
-        if (sql_end == nullptr) {
-            throw_std_runtime_error("';' not found in mysql create sql");
-        }
-        do {
-            ret = mysql_real_query(&db, sql_start, sql_end - sql_start);
+        auto sqlFilePath = config->getOption(CFG_SERVER_STORAGE_MYSQL_INIT_SQL_PATH);
+        log_debug("Loading initialisation SQL from: {}", sqlFilePath.c_str());
+        auto sql = readTextFile(sqlFilePath);
+
+        for (const auto& statement : splitString(sql, ':')) {
```
Comment (KarlStraussberger) [inline src/database/mysql/mysql_database.cc:201]: I suppose you want ; here

**Review by KarlStraussberger — CHANGES_REQUESTED** [review]

File: src/database/mysql/mysql_database.cc (hunk 1 above)
Comment (whyman) [reply src/database/mysql/mysql_database.cc:201]: Nice spot! :facepalm:

**Comment by KarlStraussberger** [comment]
We're running a bit of a security risk without hash check, but I don't see a flexible way except another config option for the hash which we default to the current value.

File: src/database/mysql/mysql_database.cc (hunk 1 above)
Comment (KarlStraussberger) [reply src/database/mysql/mysql_database.cc:201]: That's what mutual review is good for.

**Comment by whyman** [comment]
Yeah I had a think about the security aspect, but should be mostly mitigated as this SQL is only run if the DB is empty/cannot be accessed anyway.

**Pull request merged**